is-santa-coming/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Is Sana coming</title>
+</head>
+<body>
+
+
+  <script>
+
+    function isSantaComing(dateObject) {
+       var month = dateObject.getMonth();
+       var date = dateObject.getDate();
+      if (month === 12 && date === 24) {
```

Comment:
Hey Jay, Month needs to be 11 here as the months are zero indexed like arrays!

Reply:
Ok i got it. Thanks Bret